deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2

```diff
@@ -6,9 +6,18 @@
         <env name="DATABASE_PASSWORD" value="docker" />
         <env name="DATABASE_HOST" value="db" />
         <env name="INASAFE_REALTIME_BROKER_URL" value="amqp://guest:guest@{{ interpreters.inasafe_realtime.ipaddress }}:{{ docker_port_forward.inasafe_realtime.rabbitmq }}/" />
+
+        {% if inasafe_headless.use_prebuilt_docker %}
+
+        <env name="INASAFE_HEADLESS_BROKER_URL" value="amqp://guest:guest@rabbitmq:5672/" />
+
+        {% else %}
+
         <env name="INASAFE_HEADLESS_BROKER_URL" value="amqp://guest:guest@{{ interpreters.inasafe_headless.ipaddress }}:{{ docker_port_forward.inasafe_headless.rabbitmq }}/" />
+
+        {% endif %}
```

Comment:
new line here. make sure there is a blank line after every jinja conditional sentence.